firefox-plugins-installer

```diff
@@ -43,11 +43,11 @@ JAVA_LINK=${PLUGINS_DIR}/libnpjp2.so
 # previous versions are often removed from the server, so it is safest
 # for an automated script to use the stable version in the archive
 # http://fpdownload.macromedia.com/get/flashplayer/pdc/11.2.202.643/install_flash_player_11_linux.x86_64.tar.gz
-FLASH_VERSION=11.2.202.644
-FLASH_SHA256SUM=aba0e2f5940fc2588357e93f7bad84a0db51792b2f2bd1d24be9ef491e87c041
-FLASH_FILENAME=fp_${FLASH_VERSION}_archive.zip
+FLASH_VERSION=24.0.0.221
+FLASH_SHA256SUM=087048af5fa3c46a8c283403a341d4684ef4b966f4e407195b43c24bf5ae64b1
+FLASH_FILENAME=flash_player_npapi_linux.x86_64.tar.gz
```

Comment:
Not a big deal, but we use `$(arch)` for the Chrome plugin downloader, and it would probably be nice to use it here too.